lib/usd/utils/util.cpp

```diff
@@ -119,4 +120,21 @@ isInternalReference(const SdfPrimSpecHandle& primSpec)
     return isInternalRef;
 }
 
+bool 
+hasSpecs(const UsdPrim& prim)
+{
+    bool found{true};
+
+    UsdPrimCompositionQuery query(prim);
+
+    for (const auto& compQuaryArc : query.GetCompositionArcs()) {
```

Comment:
"compQuaryArc" --> "compQueryArc"

Reply:
addressed in 7fafef624444489ef3ddddd200706b9a57ed353c